veslog.md

```diff
@@ -1,5 +1,5 @@
 # Warning
-As of at least January 6, 2023 this table is no longer being supported by ITD. This means the table is not being updated and users should switch to GARFO VTR tables and ultimately CAMS when available.
+As of at least January 6, 2023 this table is no longer being supported by ITD. This means the table is not being updated and users should switch to GARFO VTR tables (Catch, Images, and Document tables in the NEFSC_GARFO schema) and ultimately CAMS when available.
```

Comment:
CATCH IMAGES and  DOCUMENT this may change to TRIP_REPORTS_CATCH
TRIP_REPORTS_DOCUMENT
TRIP_REPORTS_IMAGES but I am checking in on it

Reply:
They are both there. I don't know why.

Reply:
[bdc4063](https://github.com/NEFSC/READ-SSB-Lee-metadata/pull/27/commits/bdc4063b34c903c7fe46bd65c26132b660e5fa1f)

Reply:
I'm going to resolve this, but open an issue for later. Both of these are in the NEFSC_GARFO schema.

Reply:
added to #30. we'll sort it out when ITD does.